src/site/content/en/blog/codelab-sign-up-form-best-practices/index.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: codelab
+title: Sign-up form best practices codelab
+authors:
+  - samdutton
+scheduled: true
+date: 2020-12-01
+updated: 2020-12-01
+description: Use cross-platform browser features to build a simple sign-up form that's secure, accessible and easy to use.
+tags:
+  - forms
+  - identity
+  - layout
+  - mobile
+  - security
+  - ux
+glitch: sign-up-form-codelab-0
+glitch_path: index.html
+related_post: sign-up-form-best-practices
+---
+
+This codelab shows you how to build a sign-up form that's secure, accessible, and easy to use.
+
+## Step 1: Help password managers securely suggest and store passwords
```

Comment:
The heading and the intro sentence don't really match the actual contents of this step.

- Help password managers securely suggest and store passwords
- use form elements and attributes to make the most of built-in 
browser features

But you are introducing only `form` and `input`, not `autocomplete` and `type` attributes, and then talk about testing and accessibility.

Suggestion: Make it two steps, rename Step 1 to "Use meaningful HTML" (add links to relevant sections in the main post) and put testing and accessibility as Step 2. 

Another suggestion: Don't start with an empty Glitch, clicking view app and getting a blank screen might confuse users. Add the starter code to index.html and use Step 1 to highlight (in text) relevant bits of code and explain them.

Reply:
I've changed the codelab wording and glitch at the start to include the basic unstyled form HTML. (I gave up on codelab-0.)

I've sorted out the heading and intro for step 1 — not sure what happened there :/.

I've left testing as a subheading in step 1. This avoids the glitches not having a step 2, and I think it works OK, since this section is not about editing code.